rq_scheduler/scheduler.py

```diff
@@ -20,6 +20,7 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_TTL = 60 * 60 * 24 # 1 day
```

Comment:
Lets use here ENV_VAR or Constant to avoid the case when we have to redeploy the app if would like to change this value